gradle/plugins/src/main/kotlin/com.hedera.gradle.spotless.gradle.kts

```diff
@@ -25,7 +25,16 @@ spotless {
 
     format("misc") {
         // define the files to apply `misc` to
-        target("*.md", ".gitignore")
+        target("docs/*.md", "/*.md", "*.md", "test-clients/*.md", ".gitignore")
```

Comment:
```suggestion
        target("**/*.md", ".gitignore")
```

Reply:
Updated and resolved knock on gradle issues using Jendriks plugin suggestion